app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MoviesController < ApplicationController
+  def show  
+    @movie = Movie.find(params[:id])
+    if params[:actor_id]
```

Comment:
Looks like you are adding an actor inside of a `show` method, which doesn't quite fit with SRP & RESTful actions... Can you think of a different way to separate these concerns (showing one Movie vs. adding an actor to a movie)? 

If it were me, I would make a separate controller called `movie_actors_controller` that was only responsible for actors that belonged to a movie. There, we could have a `create` action that did what you did on lines 4-7, and leave line 3 just in this `MoviesController#show`.

Reply:
[This article](http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/) has been a helpful resource to many when figuring out how to design  Rails controllers that make sense in terms of SRP.